client/src/components/app.jsx

```diff
@@ -1,16 +1,54 @@
 import React from 'react';
+import axios from 'axios';
+import { ajax } from 'jquery'
+import CommentsList from './commentsList.jsx';
+import CommentsInputBar from './commentsInputBar.jsx'
+import Tracker from './tracker.jsx';
+import MusicProfile from './musicProfile.jsx'
 
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      comments: [],
+    };
+    this.componentDidMount = this.componentDidMount.bind(this);
```

Comment:
not sure if you need to bind component did mount?

Reply:
I didnt create a seperate function for componentdidmount to call so it needs to be bound for. now